app/components/ChartSettings/index.js

```diff
@@ -57,12 +58,19 @@ class ChartSettings extends Component {
   render() {
     return (
       <div>
-        {this._renderModule('XAxis') || ''}
-        {this._renderModule('YAxis') || ''}
-        {this._renderModule('Legend') || ''}
-        {this._renderModule('Metadata') || ''}
-        {this._renderModule('ColorPalette') || ''}
-        {this._renderCustomSettings(this.props.typeConfig) || ''}
+        <div>
+          {this._renderModule('XAxis') || ''}
```

Comment:
Instead of doing the `{ function() || ''  }` just have your function return null instead of false (react views null as `''`